modules/gapi/samples/semantic_segmentation.cpp

```diff
@@ -5,34 +5,23 @@
 #include <opencv2/gapi/operators.hpp>
 #include <opencv2/highgui.hpp>
 
+#include <opencv2/gapi/streaming/desync.hpp>
+#include <opencv2/gapi/streaming/format.hpp>
+
+#include <iomanip>
+
 const std::string keys =
     "{ h help |                                     | Print this help message }"
+    "{ desync | false                               | Desynchronize inference }"
     "{ input  |                                     | Path to the input video file }"
     "{ output |                                     | Path to the output video file }"
     "{ ssm    | semantic-segmentation-adas-0001.xml | Path to OpenVINO IE semantic segmentation model (.xml) }";
 
 // 20 colors for 20 classes of semantic-segmentation-adas-0001
-const std::vector<cv::Vec3b> colors = {
-    { 128, 64,  128 },
-    { 232, 35,  244 },
-    { 70,  70,  70 },
-    { 156, 102, 102 },
-    { 153, 153, 190 },
-    { 153, 153, 153 },
-    { 30,  170, 250 },
-    { 0,   220, 220 },
-    { 35,  142, 107 },
-    { 152, 251, 152 },
-    { 180, 130, 70 },
-    { 60,  20,  220 },
-    { 0,   0,   255 },
-    { 142, 0,   0 },
-    { 70,  0,   0 },
-    { 100, 60,  0 },
-    { 90,  0,   0 },
-    { 230, 0,   0 },
-    { 32,  11,  119 },
-    { 0,   74,  111 },
+static std::vector<cv::Vec3b> colors = {
+    {0, 0, 0},       {0, 0, 128},     {0, 128, 0}, {0, 128, 128}, {128, 0, 0},   {128, 0, 128}, {128, 128, 0},
+    {128, 128, 128}, {0, 0, 64},      {0, 0, 192}, {0, 128, 64},  {0, 128, 192}, {128, 0, 64},  {128, 0, 192},
+    {128, 128, 64},  {128, 128, 192}, {0, 64, 0},  {0, 64, 128},  {0, 192, 0},   {0, 192, 128}, {128, 64, 0}
 };
```

Comment:
What's the reason for this change? Previously it was just one line per class, easy to edit.

Reply:
No problem, I'll keep old alignment